.github/workflows/mend.yml

```diff
@@ -0,0 +1,18 @@
+name: "mend"
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
Shall we drop this?

Reply:
Yeah thats fair, pull request should be sufficient. 👍